src/go/k8s/tests/e2e/centralized-configuration-drift/01-external-change.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: external-change
```

Comment:
NIT: I don't see the assert for this job. It is for purpose? Is there a possibility that `get-centralized-config-1` job finish before this container execute `segment_appender_flush_timeout_ms` config change.

Reply:
True, thanks for noticing :smile: